src/meta/meta_ckpt.c

```diff
@@ -1262,6 +1262,195 @@ __wt_meta_ckptlist_update_config(
     return (ret);
 }
 
+/*
+ * __ckpt_extract_blkmod_bitmap --
+ *     Extracts the blkmod info in a checkpoint into a bitmap in a WT_ITEM.
+ */
+static int
+__ckpt_extract_blkmod_bitmap(
+  WT_SESSION_IMPL *session, WT_CKPT *ckpt, const char *id, WT_ITEM *output_bitmap)
+{
+    WT_BLOCK_MODS *blk;
+    u_int i;
+
+    WT_CLEAR(*output_bitmap);
+
+    for (i = 0, blk = &ckpt->backup_blocks[0]; i < WT_BLKINCR_MAX; ++i, ++blk)
+        if (F_ISSET(blk, WT_BLOCK_MODS_VALID) && strcmp(id, blk->id_str) == 0)
+            WT_RET(__wt_buf_set(session, output_bitmap, blk->bitstring.data, blk->bitstring.size));
+
+    return (0);
+}
+
+/*
+ * __get_blkmods --
+ *     Extracts the blkmod info from the metadata about a file+id into a bitmap in a WT_ITEM.
```

Comment:
Same comment here regarding the description. You can leave it as is if you decide to ignore my first comment.

Reply:
See above.